cdc/sinkv2/eventsink/txn/worker.go

```diff
@@ -86,8 +96,11 @@ func (w *worker) runBackgroundLoop() {
 }
 
 func (w *worker) doFlush() bool {
-	if err := w.backend.Flush(); err != nil {
-		// TODO: handle err.
+	// TODO: support to cnacel the worker in flushing.
```

Comment:
why not return true when flushed successfully?

Reply:
It returns true if it's going to exit the background goroutine. It's updated to be more friendly for reviewers.